### DISCUSSION TITLE
Mobile version of Beaker Browser

### DISCUSSION THREAD

**Discussion by ONLYA** [beakerbrowser/beaker#1853] [Ideas] [answered]
I think that a mobile Beaker Browser will be awesome. Therefore, anyone can browse the hyperdrive through their phone. I can even find scenarios that may be applied: Text, Voice or Video chat with others on-the-go, sharing files and text with computers.

Do you guys have that kind of plan in the future?

**Comment by RangerMauve** [comment, marked as answer]
There's a mobile browser for iOS and Android being made by @Calm-Rad which supports `hyper://` and most of Beakers APIs.

https://gitlab.com/gateway-browser/gateway
